### PR TITLE
Added feature to import examples automatically in the webapp

### PR DESCRIPTION
**Pull request by onlinecco**
only toyexample,drugresponse,emailspam and edison feature work.

### CODE REVIEW

**Comment by kordjamshidi** [comment]
Great, thanks! 
(it seems there is a conflict)

File: saul-webapp/public/javascripts/main.js [hunk 1]
```diff
@@ -8,10 +8,62 @@ $(document).ready(function(){
...
+        data: JSON.stringify({
+            "data": ""
+        }),
+        success : addButtons,
+        error: onError
+    });
+
+
+}
 var setupExamples = function(){
```
Comment (danyaljj) [inline saul-webapp/public/javascripts/main.js]: Do we still need to keep `setupExamples`?

**Comment by danyaljj** [comment]
One minor comment; otherwise good to merge!

**Comment by onlinecco** [comment]
@danyaljj @kordjamshidi  can anyone resolve the conflict and merge this? It seems that OSX is not sensitive to filenames' cases, and it cant detect a conflict on my machine

I simply changed first character of the filename to uppercase to match the object name. 
spamClassifiers.scala => SpamClassifiers.scala
spamDataModel.scala => SpamDataModel.scala

**Comment by danyaljj** [comment]
> It seems that OSX is not sensitive to filenames' cases, and it cant detect a conflict on my machine 

@onlinecco that is true. There is a simple solution for that. Make an intermediate change which involves non-case change. For example to rename `name` to `Name`: 

`name` -> `name2` -> `Name`

**Comment by bhargav** [comment]
Looks good to me! Can merge if conflicts are resolved and Semaphore is happy.

**Comment by kordjamshidi** [comment]
@onlinecco : see @danyaljj 's comment, that will work. I also have the same problem with Mac.

**Pull request merged**